src/electron/views/device-connect-view/components/android-setup/default-android-setup-components.ts

```diff
@@ -26,5 +26,7 @@ export const defaultAndroidSetupComponents: AndroidSetupStepComponentProvider =
     'installing-service': InstallingServiceStep,
     'detect-devices': DetectDevicesStep,
     'detect-service': DetectServiceStep,
+    'configuring-port-forwarding': DetectAdbStep,
```

Comment:
These lines need to point to the correct steps. :)

Reply:
Placeholder values until Dan merges the steps

Reply:
These have real values now